openapi.json

```diff
@@ -263,6 +263,103 @@
         ]
       }
     },
+    "/organizations/{orgId}/clusters/{clusterId}/users/{userId}/report/{ruleID}": {
```

Comment:
Don't we already have it in https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/openapi.json#L355

Reply:
Yes, but in smart proxy. Now we need to have endpoint for single rule in aggregator.

Reply:
@bond95 I successfully fetched a report for a single rule from CI and QA. Why is this PR needed now if everything works already?

Reply:
It's better to follow the convention and use `/rules/{ruleId}`